ckan/config/middleware/flask_app.py

```diff
@@ -181,6 +182,16 @@ def hello_world():
     def hello_world_post():
         return 'Hello World, this was posted to Flask'
 
+    # WebAssets
+    public_folder = config.get('ckan.base_public_folder')
+    webassets_folder = os.path.join(
+        os.path.dirname(ckan.__file__), public_folder, 'webassets'
```

Comment:
does this mean that webassets need to be stored under the ckan source directory? That feels wrong, especially for package installs.

Reply:
Yes, I agree. I just chose the most simple way of serving files. But it may be actually served from any other place in the filesystem(like storage directory) as long as Flask has read access to it. We also may build assets before starting the server and serve them separately(not via flask), but it will add some configuration overhead. 
What you think about putting compiled assets into a storage folder? In that way, we avoid writing into CKAN filesystem, but storage directory wasn't required previously, so we need to change the installation process. Or, maybe there are other suggestions?

Reply:
Finally, got some time for update. I'll use additional configuration variable `ckan.webassets.path` (with fallback to `ckan.storage_path` + `webassets` and RuntimeError if neither is specified) So, they can be placed near the resources and group images - it's reasonable enough, i suppose